packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -14,7 +14,9 @@ import codecommit = require('@aws-cdk/aws-codecommit');
 
 const repo = new codecommit.Repository(this, 'MyRepo', { repositoryName: 'foo' });
 new codebuild.Project(this, 'MyFirstCodeCommitProject', {
-    source: new codebuild.CodeCommitSource(repo)
+    source: new codebuild.CodeCommitSource({
```

Comment:
The indentation here is off. Our standard these days is 2 spaces.

This is probably true for all files?

Reply:
Yes, none of the ReadMe files were changed to 2 spaces when the indentation was changed for code, hence I'm sticking to 4 here. We should probably do a separate pass over the Readmes to fix them all.